stock_picking_dispatch/wizard/check_assign_all.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# © 2012-2014 Guewen Baconnier, Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import _, models, api
+from openerp.exceptions import UserError
+
+
+class CheckAssignAll(models.TransientModel):
+    _name = 'picking.dispatch.check.assign.all'
```

Comment:
Why not a button in the main screen with a `confirm` attribute if you need a text with a confirmation screen?